.changeset/floppy-news-buy-1.md

```
@@ -0,0 +1,38 @@
+---
+"task-master-ai": minor
+---
+
+Add new `scope-up` and `scope-down` commands for dynamic task complexity adjustment
+
+This release introduces two powerful new commands that allow you to dynamically adjust the complexity of your tasks and subtasks without recreating them from scratch.
+
+**New CLI Commands:**
+- `task-master scope-up` - Increase task complexity (add more detail, requirements, or implementation steps)
+- `task-master scope-down` - Decrease task complexity (simplify, remove unnecessary details, or streamline)
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)